raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientReconfigTest.java

```diff
@@ -2085,8 +2091,8 @@ void testFollowerSendsUpdateVoter() throws Exception {
             .withLocalListeners(localListeners)
             .build();
 
-        // waiting for 3 times the fetch timeout sends an update voter
-        for (int i = 0; i < 3; i++) {
+        // waiting for FETCH request until the UpdateRaftVoter request is set
+        for (int i = 0; i < NUMBER_FETCH_TIMEOUTS_IN_UPDATE_PERIOD; i++) {
```

Comment:
is there any point to this for loop if `NUMBER_FETCH_TIMEOUTS_IN_UPDATE_PERIOD` is statically set to 1?

Reply:
No but that's because I changed the update voter timer from `3 * fetch timeout` to just `fetch timeout`. I figured that I should leave it parametrized incase we later decide to change the update voter timer.